src/components/picture/index.stories.mdx

```diff
@@ -0,0 +1,145 @@
+import { Meta, Story, Canvas, ArgsTable } from "@storybook/addon-docs";
+
+import Picture from ".";
+import Source from "./_children/source";
+import Image from "../image";
+
+<Meta title="Components/Picture" component={Picture} />
+
+# Picture
+
+The `Picture` component and `Picture.Source` subcomponent render the HTML5 `<picture>` and `<source>` tags, respectively. Similar to the Image, the Picture component displays images as both a means of content and content support. Picture, however, allows for different images (with respective resize options) to be used explicitely for different media queries. This allows for different Art Direction approaches for specific viewport dimensions.
+
+For `resizedOptions`, here are the query parameters that are available:
+https://github.com/WPMedia/arc-photo-resizer/tree/main/docs/usage#query-parameters
+
+Crucially, an "auth" key will allow the image to be resized. This object will resize the image to the specified dimensions and other properties given in the query parameters in the `src`.
+
+The `Picture` component can use zero or more `Picture.Source` subcomponents and should also include one `Image` as a supportive fallback. Those are the two types of children a Picture will support.
+
+## Usage
+
+```jsx
+import { Picture, Image } from "@wpmedia/arc-themes-components";
+
+const Feature = () => <Picture>
+	<Picture.Source
+		src="cat.jpg"
+		width={400}
+		height={300}
+		media="(min-width: 768px)"
+		resizerURL="https://resizer.com/v2/"
+		resizedOptions={{
+			auth: "secret",
+		}}
+	<Image
+		src="cat.jpg"
+		width={200}
+		height={200}
+		resizerURL="https://resizer.com/v2/"
+		alt="A cat meowing"
```

Comment:
Should this have the resizer options as well, err... `resizedOptions` (auth is required, no?)

Reply:
Good call! Thanks!